src/Analyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer/DurableFunctionsAnalyzer.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard1.3</TargetFramework>
```

Comment:
We've been targeting `netstandard2.0` for our projects, so I suggest we do the same here for consistency.